src/GitHub.Exports/Services/VSServices.cs

```diff
@@ -68,6 +73,49 @@ public void ActivityLogWarning(string message)
             }
         }
 
+        /// <summary>
+        /// There doesn't appear to be a command that directly opens a target repo.
+        /// Our workaround is to create, open and delete a solution in the repo directory.
+        /// This triggers an event that causes the target repo to open. ;)
+        /// </summary>
```

Comment:
Small nit here, but a `<summary>` is supposed to be a one-sentence description of what the method does. This comment is good, but IMO it should be in the `<remarks>` section.